lib/src/install.rs

```diff
@@ -1229,7 +1223,12 @@ async fn install_to_filesystem_impl(state: &State, rootfs: &mut RootSetup) -> Re
             .context("Writing aleph version")?;
     }
 
-    crate::bootloader::install_via_bootupd(&rootfs.device, &rootfs.rootfs, &state.config_opts)?;
+    if cfg!(target_arch = "s390x") {
+        // TODO: Integrate s390x support into install_via_bootupd
```

Comment:
Yeah...that would definitely be better in the longer term I think, but this is totally fine to do here for now. See also https://github.com/coreos/bootupd/issues/432